gradient/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p input
+if [ ! -e "input/UK-dem-50m-4326.tif" ]; then
+	wget https://play.abstreet.org/dev/data/input/shared/elevation/UK-dem-50m-4326.tif.gz -P input
```

Comment:
I'll describe this file in the frontend PR. It was generated from https://www.ordnancesurvey.co.uk/products/os-terrain-50 by Malcolm Morgan. I've been using it for a few years in A/B Street things

Reply:
For posterity, https://github.com/mem48/OTPR/blob/master/scripts/prep_DEM.R was the script to make the raster